src/api/transformArticleApi.ts

```diff
@@ -134,10 +134,21 @@ export const transformArticle = async (
 
   let footnoteCount = 0;
   const embedPromises = await Promise.all(
-    embeds.map((embed, index) => {
+    embeds.map(async (embed, index) => {
       if (embed.data.resource === "footnote") {
         footnoteCount += 1;
       }
+      if (embed.data.resource === "uu-disclaimer") {
+        const transformedContent = await transformArticle(embed.data.disclaimer, context, undefined, {});
```

Comment:
Hva skjer hvis denne tryner?

Reply:
Det vil være lik oppførsel som ellers, tror jeg, dersom gql feiler så vil vi vise feilmeldings-side i ndla-frontend

Reply:
Da lurer jeg på om v i heller burde gå for en try-catch!

Reply:
Hva med noe sånt?

Reply:
Ser greit ut det. Har NDLA/Hedvig sagt noe om hva som skal skje ved eventuelle feil? Skal vi forsøke å vise innholdet, eller skal vi bare droppe alt?